torch/_C/_autograd.pyi

```diff
@@ -14,8 +14,18 @@ class ProfilerConfig:
     def __init__(self, state: ProfilerState, report_input_shapes: bool, profile_memory: bool) -> None: ...
     ...
 
+# Defined in torch/csrc/autograd/profiler.h
+class EventKind:
```

Comment:
I'm not sure this is right, because `EventKind` doesn't exist as a Python object in `torch._C._autograd`. I don't see any other `C10_API_ENUM`'s that are stubbed like this. Is this necessary?

Reply:
Yes, you're right. For some reason I thought this enum was used in python.